apps/app/utils/postWishLinkFromShare.ts

```
@@ -15,17 +31,27 @@ const postWishLink = async (productUrl: string, accessToken: string) =>
     body: JSON.stringify({ url: productUrl }),
   });
 
+/** 실패 응답 body 의 code 만 뽑는다. 파싱 실패는 무시 — 사유는 status 로 이미 갈렸다. */
+const readErrorCode = async (response: Response) => {
+  try {
+    const body = (await response.json()) as { code?: string | null };
+    return body.code ?? undefined;
```

ESLint

[error] Unexpected use of undefined.

(no-undefined)